client/extensions/helloWorld2/npm-debug.log

```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```

Comment:
this file shouldn't be there we can probably ignore `*.log`

Reply:
`npm-debug.log` shouldn't be committed either

Reply:
my bad, I did not notice it somehow..